left_wheel_joint: "wheel_left_joint"
right_wheel_joint: "wheel_right_joint"
vx_noise: 0.005
w_noise: 0.01
slip_max: 0.15
slip_min: 0.1
ox: [2, -2, -0.5, 1, 0.6, 1.2, 0]
oy: [2, 1.5, 0.2, 0.8, -0.3, -1, -0.5]
obs_var: [0.03,0.03,0.01]
obs_rad: 0.076
num_tubes: 2.0
robot_radius: 0.1
lidar_min_range: 0.12
lidar_max_range: 3.5
lidar_dtheta: 1
lidar_n_samples: 360
lidar_resolution: 0.015
lidar_noise: 0.005
wall_width: 6
wall_height: 6

odom_frame_id: "odom"
body_frame_id: "base_footprint"